src/platformdirs/macos.py

```diff
@@ -42,8 +56,21 @@ def user_cache_dir(self) -> str:
 
     @property
     def site_cache_dir(self) -> str:
-        """:return: cache directory shared by users, e.g. ``/Library/Caches/$appname/$version``"""
-        return self._append_app_name_and_version("/Library/Caches")
+        """
+        :return: cache directory shared by users, e.g. ``/Library/Caches/$appname/$version``.
+          If we're using a Python binary managed by `Homebrew <https://brew.sh>`_, the directory
+          will be under the Homebrew prefix, e.g. ``/opt/homebrew/var/cache/$appname/$version``.
+          If `multipath <platformdirs.api.PlatformDirsABC.multipath>` is enabled and we're in Homebrew,
+          the response is a multi-path string separated by ":", e.g.
+          ``/opt/homebrew/var/cache/$appname/$version:/Library/Caches/$appname/$version``
+        """
+        path_list = [self._append_app_name_and_version("/Library/Caches")]
+        is_homebrew = sys.prefix.startswith("/opt/homebrew")
+        if is_homebrew:
+            path_list.insert(0, self._append_app_name_and_version("/opt/homebrew/var/cache"))
+        if self.multipath:
+            return os.pathsep.join(path_list)
+        return path_list[0]
```

Comment:
Rather than inserrt start with homebrew, and use append for the self._append_app_name_and_version